test/libsolidity/semanticTests/strings/string_escapes.sol

```diff
@@ -0,0 +1,10 @@
+contract test {
+    function f() public pure returns (bytes32) {
+        bytes32 escapeCharacters = "\t\n\r\'\"\\";
```

Comment:
Something is wrong though, this semantic test expectedly breaks if I add `\f`, `\b` or `\v` -- but somehow `syntaxTests/strings/string_escapes.sol` passes.

Reply:
> Something is wrong though, this semantic test expectedly breaks if I add `\f`, `\b` or `\v` -- but somehow `syntaxTests/strings/string_escapes.sol` passes.

Are you sure? CI has it fail as expected... if locally for you that's different, that's weird - I don't see anything that could be osx specific in any of that...

Reply:
PEBKAC. `isoltest -t syntaxTests/strings*` did not work, because the directory is named `syntaxTests/string` and `semanticTests/strings`. Singular vs plural.

Reply:
Haha, I was literally just watching ``t_osx_soltest``, hardly believing that it just wouldn't fail - but apparently by chance it's just one of the last tests, it runs, so ``t_osx_soltest`` indeed also fails on that syntax tests just as the other runs do...